app/src/main/java/com/kickstarter/ui/activities/SettingsNewActivity.kt

```diff
@@ -0,0 +1,18 @@
+package com.kickstarter.ui.activities
+
+import android.databinding.DataBindingUtil
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import com.kickstarter.R
+import com.kickstarter.databinding.SettingsLayoutNewBinding
+import com.kickstarter.viewmodels.SettingsNewViewModel
+
+class SettingsNewActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val binding: SettingsLayoutNewBinding = DataBindingUtil.setContentView(this, R.layout.settings_layout_new)
+
+        binding.viewModel = SettingsNewViewModel(this)
+    }
+}
```

Comment:
Let's throw a new line here. When that weird little circle with slash and arrow icon is there, that means you need to add a new line.